ups/eupspkg.cfg.sh

```diff
@@ -44,12 +44,15 @@ install()
         # Install packages on which the stack is known to depend
 
         export PATH="$PREFIX/bin:$PATH"
-        local tmpdir=${TMPDIR:-/tmp}
         local reqfile='conda_packages.txt'
         local baseurl='https://raw.githubusercontent.com/lsst/lsstsw/master/etc/'
-        (cd "$tmpdir"; $CURL -# -L -O "${baseurl}/${reqfile}")
+        local tmpfile
+        tmpfile=$(mktemp -t "${reqfile}.XXXXXXXX")
```

Comment:
on OS X the `-t` option specifies a fixed prefix and these `X`s are not expanded (the randomized bit is appended to the `XXXXX`). I think you mean to be using:
```shell
mktemp -t "${reqfile}" XXXXXXX
```
Although I don't think the `X`s are needed really.